### ISSUE TITLE
Cannot start Jellyfin server because of missing libc.musl-x86_64.so.1library

### ISSUE THREAD

**Issue by SBado** [jellyfin/jellyfin#5594]
**Describe the bug**
Starting from version 1.0.7.1, I cannot start Jellyfin server because it now requires libc.musl-x86_64.so.1 (which is missing in my system).

**System (please complete the following information):**
 - OS: openSUSE Tumbleweed 
 - Jellyfin Version: 10.7.1 (combined)

**To Reproduce**
1. Download the archive from [here](https://repo.jellyfin.org/releases/server/linux/stable/combined/jellyfin_10.7.1.tar.gz)
2. Unpack it
3. `cd` to `jellyfin_10.7.1/`
4. Run `./jellyfin`
5. Error `bash: ./jellyfin: No such file or directory` will appear
6. Run `ldd` on `jellyfin` executable
7. It will say `libc.musl-x86_64.so.1 => not found`

**Expected behavior**
Jellyfin starts normally

**Logs**
N/A

**Screenshots**
N/A

**Additional context**
This is the complete `ldd` output:
```
        linux-vdso.so.1 (0x00007fff7effc000)
        libstdc++.so.6 => /usr/lib64/libstdc++.so.6 (0x00007fb553e3c000)
        libgcc_s.so.1 => /lib64/libgcc_s.so.1 (0x00007fb553e21000)
        libc.musl-x86_64.so.1 => not found
        libm.so.6 => /lib64/libm.so.6 (0x00007fb553cdc000)
        libc.so.6 => /lib64/libc.so.6 (0x00007fb553b12000)
        /lib/ld-musl-x86_64.so.1 => /lib64/ld-linux-x86-64.so.2 (0x00007fb55408a000)
```

This is the output for Jellyfin version version 10.7.0 (downloaded from [here](https://repo.jellyfin.org/releases/server/linux/versions/stable/combined/10.7.0/jellyfin_10.7.0.tar.gz) and working fine):
```
        linux-vdso.so.1 (0x00007ffe1d972000)
        libpthread.so.0 => /lib64/libpthread.so.0 (0x00007f148f59b000)
        libdl.so.2 => /lib64/libdl.so.2 (0x00007f148f594000)
        libstdc++.so.6 => /usr/lib64/libstdc++.so.6 (0x00007f148f37b000)
        libm.so.6 => /lib64/libm.so.6 (0x00007f148f236000)
        libgcc_s.so.1 => /lib64/libgcc_s.so.1 (0x00007f148f21b000)
        libc.so.6 => /lib64/libc.so.6 (0x00007f148f051000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f148f7fd000)
```

**Comment by anozdba** [comment]
Problem still exists in 10.7.2

jellyfin@mediaServer:/opt/jellyfin/jellyfin> ldd jellyfin
        linux-vdso.so.1 (0x00007ffd2f9fa000)
        libstdc++.so.6 => /usr/lib64/libstdc++.so.6 (0x00007f2405d2e000)
        libgcc_s.so.1 => /lib64/libgcc_s.so.1 (0x00007f2405b15000)
        libc.musl-x86_64.so.1 => not found
        libm.so.6 => /lib64/libm.so.6 (0x00007f24057dd000)
        libc.so.6 => /lib64/libc.so.6 (0x00007f2405422000)
        /lib/ld-musl-x86_64.so.1 => /lib64/ld-linux-x86-64.so.2 (0x00007f2406108000)

**Comment by clyvari** [comment]
So, for anyone that want to upgrade to 10.7.x but can't because of this issue:
It seems the `combined` release of jellyfin now uses the `musl` build for the server, however the "standard" server release still exists.

Question for @anthonylavado (since you confirmed the bug):
Is this the new intended behavior that the `combined` release uses the `musl` server build?

### The workaround:
Check here: https://repo.jellyfin.org/releases/server/linux/stable/
`combined/jellyfin_10.7.2.tar.gz` --> ❌ Uses musl library, you don't want this|
`server/jellyfin-server_10.7.2_linux-amd64-musl.tar.gz` -->❌ This is what is packaged inside the combined release
`server/jellyfin-server_10.7.2_linux-amd64.tar.gz` -->✅ This is probably what was packaged into the combined version before, and what you want now

The combined release is just the `server` release + the `web` release inside (`jellyfin_10.7.2/jellyfin-web`)

If you have already downloaded the 10.7.x combined release, you just need to
* Download the `server/jellyfin-server_10.7.2_linux-amd64.tar.gz` file
* Copy the files inside to where you jellyfin executable files usually are

**Comment by sevenrats** [comment]
Much of this has been refactored for 10.8 please open a new issue if you still experience this.

**Comment by clyvari** [comment]
@sevenrats as far as I know, this is not an issue anymore. I'm unsure of when it was fixed, but it was caused by how the archive was created, see the referencing issue above.

It was introduced by the recent (at the time) addition of a muscl build, but it was corrected and hasn't been an issue, at least for 10.8 and up
